app/models/shelter.rb

```diff
@@ -1,4 +1,11 @@
 class Shelter < ApplicationRecord
   has_many :pets, dependent: :destroy
   has_many :reviews, dependent: :destroy
+
+  def pets_pending?
+    pets.any? do |pet|
+      pet.adoption_status == "pending"
+    end
+  end
+
```

Comment:
HJelper method to determine if any pets have had application approved. I am not sure if this is considered ruby code and needs to be refactored to Active Record. Let me know what you think.

Reply:
I am not sure either. That syntax seems to be ruby; let's check in with the instructors. I think we'll have to change it.